History.md

```diff
@@ -1,4 +1,3 @@
-
```

Comment:
Removed entry for now because publishing will create a new entry with commit messages and we can update that entry to be more accurate